### ISSUE TITLE
add Google satellite as alternative base map 

### ISSUE THREAD

**Issue by gubuntu** [kartoza/WRODataPlatform#29]
In the map view, OSM is the default and only basemap.

The WRO team requested that Google Satellite be added as an alternative base map. 

Please get a key for Google Maps via the WRO GCP account to enable this.

**Comment by Mohab25** [comment]
@mikev3003 @Christiaan34 added a google maps base map with satellite view for bounds capture, looking to generalize the same map for the whole site. (the following gif is compressed twice to be uploaded to github, the actual spatial resolution of google satellite view is quite good)
![google maps for bounds capture (1) (1)](https://user-images.githubusercontent.com/58084234/177023344-83f4a0bb-7466-42c7-a403-5869d11e4553.gif)

**Comment by gubuntu** [comment]
adding a google base map doesn't mean switching to the google maps api, just adding the base map to the current map (which is / was Leaflet, right?)

**Comment by Mohab25** [comment]
@gubuntu we need to be able to draw over the map to capture boundaries, it's a part of the api

**Comment by gubuntu** [comment]
yes but you can also draw in Leaflet, why change the implementation? We generally avoid using the Google Maps or other proprietary APIs. In this case with the WRO on GCP, there might be an argument to be made to use
